controllers/part/part_ctlr.go

```diff
@@ -77,8 +79,45 @@ func All(w http.ResponseWriter, r *http.Request, params martini.Params, enc enco
 			count = ct
 		}
 	}
+	//Convert given dates to ISO8601 format.
+	if qs.Get("modified-from") != "" {
+		from = qs.Get("modified-from")
+		_, err := time.Parse(time.RFC3339, from)
+		if err != nil {
+			_, err := time.Parse(time.RFC3339, from+"-06:00")
```

Comment:
Would it be better just to send an invalid request if the date format is malformed?
I think it would be weird if someone had a typo for pacific time and then they all of a sudden got central time back.